docs/source/modin/supported/index_supported.rst

```diff
@@ -159,7 +159,8 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``searchsorted``            | N                               |                                  |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``sort_values``             | N                               |                                  |                                                    |
+| ``sort_values``             | P                               | key                              | Snowpark pandas currently uses stable sort when    |
+|                             |                                 |                                  | sorting the index values. Pandas uses quicksort.   |
```

Comment:
Does this comment also apply to `Series.sort_values` and `DataFrame.sort_values`? If so, then we should update their docs too. It can be in a separate PR though.

Reply:
Yeah! Should apply there as well, I'll make a separate PR for this

Reply:
https://github.com/snowflakedb/snowpark-python/pull/1916/files
Here's the PR!